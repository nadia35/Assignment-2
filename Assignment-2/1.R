addNumbers <- function(x,y){
  myNums <- c(x,y)
  return(sum(myNums, na.rm = TRUE))
}
