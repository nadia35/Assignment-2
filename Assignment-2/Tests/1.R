source("1.R")

print(addNumbers(2, 2))
print(addNumbers(-2, 2))
print(addNumbers(20.2, 8.22822))
print(addNumbers(0, 0))
print(addNumbers(NA, 2))
print(addNumbers(2, NA))