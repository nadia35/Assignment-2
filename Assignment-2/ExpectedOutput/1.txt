[1] 4
[1] 0
[1] 28.42822
[1] 0
[1] 2
[1] 2
